From 2cd44069945ff14f59a5b48439d46dac0dfb4035 Mon Sep 17 00:00:00 2001
From: Josh Blum <josh@joshknows.com>
Date: Sun, 1 Apr 2012 22:12:47 -0700
Subject: [PATCH 48/49] examples: fix bsd compilation for network relay example

Fixes compilation on platforms that dont have one of the OS defines
by providing #else case for rx_dsp_buff_size constant.
This also makes OSX the special case and assumes other OS
can handle the large RX socket buffer size.
---
 host/examples/network_relay.cpp |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/host/examples/network_relay.cpp b/host/examples/network_relay.cpp
index a7fdb7e..e8f9e66 100644
--- a/host/examples/network_relay.cpp
+++ b/host/examples/network_relay.cpp
@@ -1,5 +1,5 @@
 //
-// Copyright 2010-2011 Ettus Research LLC
+// Copyright 2010-2012 Ettus Research LLC
 //
 // This program is free software: you can redistribute it and/or modify
 // it under the terms of the GNU General Public License as published by
@@ -34,10 +34,10 @@ static const size_t insane_mtu = 9000;
 
 boost::mutex spawn_mutex;
 
-#if defined(UHD_PLATFORM_MACOS) || defined(UHD_PLATFORM_BSD)
+#if defined(UHD_PLATFORM_MACOS)
     //limit buffer resize on macos or it will error
     const size_t rx_dsp_buff_size = size_t(1e6);
-#elif defined(UHD_PLATFORM_LINUX) || defined(UHD_PLATFORM_WIN32)
+#else
     //set to half-a-second of buffering at max rate
     const size_t rx_dsp_buff_size = size_t(50e6);
 #endif
-- 
1.7.2.5

